frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -316,36 +324,31 @@ export const getInternalImageInitialValues = (editAppResource: K8sResourceKind)
     'metadata.labels["app.openshift.io/runtime-version"]',
     '',
   );
-  const deployImageInitialValues = {
-    searchTerm: '',
+  return {
+    ...deployImageInitialValues,
     registry: 'internal',
     imageStream: {
       image: imageStreamName,
       tag: imageStreamTag,
       namespace: imageStreamNamespace,
     },
-    isi: {
-      name: '',
-      image: {},
-      tag: '',
-      status: { metadata: {}, status: '' },
-      ports: [],
-    },
-    image: {
-      name: '',
-      image: {},
-      tag: '',
-      status: { metadata: {}, status: '' },
-      ports: [],
-    },
-    build: {
-      env: [],
-      triggers: {},
-      strategy: '',
+  };
+};
+
+export const getExternalImagelValues = (appResource: K8sResourceKind) => {
+  const name = _.get(appResource, 'spec.template.spec.containers[0].image', null);
+  if (_.isEmpty(appResource) || !name) {
+    return {};
+  }
+  return {
+    ...deployImageInitialValues,
+    searchTerm: name,
+    registry: 'external',
```

Comment:
can we change this to RegistryType.External
```suggestion
    registry:RegistryType.External,
```

Reply:
Done